src/config/user.json

```
@@ -1,3 +1,3 @@
 {
-  "username": ""
+  "aladdin": ""
```

Please change this back to `username`, otherwise, the program won't be able to deploy successfully.
```suggestion
  "username": ""
```